pydm/widgets/logdisplay.py

```diff
@@ -22,7 +22,7 @@
 logger = logging.getLogger(__name__)
 
 
-def logger_destroyed(log, obj):
+def logger_destroyed(log):
```

Comment:
Thanks! Can remove lines 34-36 as well:

`    obj : QWidget
        The widget which is being destroyed. All childs of this widget are
        already destroyed and must not be accessed.`